css-canvas-medical | Nick Hernandez

Nickolas-Hernandez opened the pull request:
![Screen Shot 2021-01-23 at 4 59 29 PM](https://user-images.githubusercontent.com/75342275/105618694-1ba65180-5d9f-11eb-9435-485d1af5a1ba.png)
![Screen Shot 2021-01-23 at 4 59 50 PM](https://user-images.githubusercontent.com/75342275/105618697-206b0580-5d9f-11eb-85b9-54049b82c21f.png)
![Screen Shot 2021-01-23 at 4 59 59 PM](https://user-images.githubusercontent.com/75342275/105618698-2234c900-5d9f-11eb-9b2c-1d1de7f3ae7e.png)

thebearingedge commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,202 @@
+.links-and-location-section {
+  background-color: #525351;
+  color: white;
+  height: 40vh;
```
This is going to seem random, but 99.999% _never_ use vh for hardly anything. Favor `px` for height if it works.

thebearingedge commented on css-canvas-medical/styles.css:
```diff
@@ -0,0 +1,202 @@
+/* Globals and Layouts */
+html {
+  font-family: "Avenir", sans-serif;
+}
```
Typically we put this on `body` and not `html`, since `body` is the only thing that is visible.

thebearingedge reviewed (changes requested):
Fantasic, just a couple minor tweak.

Nickolas-Hernandez commented:
![Screen Shot 2021-01-28 at 2 47 16 PM](https://user-images.githubusercontent.com/75342275/106208125-baec8f80-6177-11eb-9b1c-b36ce242052c.png)


Added font-family property to body selector, and changed all vh values to px.
